mock/mock_test.go

```diff
@@ -1221,6 +1222,45 @@ func Test_MockReturnAndCalledConcurrent(t *testing.T) {
 	wg.Wait()
 }
 
+type timer struct{ Mock }
+
+func (s *timer) GetTime(i int) string {
+	return s.Called(i).Get(0).(string)
+}
+
+func TestAfterTotalWaitTimeWhileExecution(t *testing.T) {
```

Comment:
These tests slow down the test run by ˜3 seconds.

Can you reduce wait times a lot more to keep tests quick?